dashboard/src/components/settings/signInMethods/ConfirmProvidersUpdatedSettings/ConfirmProvidersUpdatedSettings.tsx

```diff
@@ -0,0 +1,60 @@
+import { useDialog } from '@/components/common/DialogProvider';
+import { Alert } from '@/ui/Alert';
+import Button from '@/ui/v2/Button';
+import ArrowSquareOutIcon from '@/ui/v2/icons/ArrowSquareOutIcon';
+import Link from '@/ui/v2/Link';
+import Text from '@/ui/v2/Text';
+import { useState } from 'react';
+
+export default function ConfirmProvidersUpdatedSettings() {
+  const { openDialog } = useDialog();
+  const [visible, setVisible] = useState(true);
+
+  const removeElement = () => {
+    setVisible((prev) => !prev);
+  };
+
+  function handleConfirmDialog() {
+    openDialog('CONFIRM_PROVIDERS_UPDATED', {
+      title: 'Confirm updated URLs for all providers?',
+      payload: {
+        onSubmit: () => removeElement(),
+      },
+    });
+  }
+
+  if (!visible) {
+    return null;
+  }
+
+  return (
+    <Alert className="grid grid-flow-col px-4 place-content-between bg-amber-500">
```

Comment:
paddings and spacings are a little bit off here:

<img width="800" alt="Screenshot 2022-11-30 at 8 39 30" src="https://user-images.githubusercontent.com/310881/204743592-ae9066b3-c2a5-4ec0-ae7e-05466862e936.png">